pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -137,6 +137,8 @@ type ShootSpec struct {
 	// This field is immutable.
 	// +optional
 	SchedulerName *string `json:"schedulerName,omitempty" protobuf:"bytes,21,opt,name=schedulerName"`
+	// CloudProfile is a reference to a CloudProfile or a NamespacedCloudProfile.
```

Comment:
optional tag missing

Reply:
Added in https://github.com/gardener/gardener/pull/9440/commits/8daad6c29ddd3ced99d036dbd0bc5d77cf3c9369